feat(genapi): update continue intellij idea tutorial (#5355)
scaleway/docs-content: pages/generative-apis/reference-content/adding-ai-to-intellij-using-continue.mdx

- Click **Continue** in the menu on the right. .
- In the prompt section, click on **Select model** dropdown, then on **Add Chat model**.
- Select **Scaleway** as provider.
- Select the model you want to use (we recommend `Qwen 2.5 Coder 32b` to get started with chat and autocompletion only).
- Enter your **Scaleway secret key**.
    <Message type="tip">
     To start with, we recommend you use a Scaleway secret key having access to your `default` Scaleway project.
    </Message>

These actions will edit automatically your `config.yaml` file. To edit it manually, see [Configure Continue through configuration file](#configure-continue-through-configuration-file).

<Message type="note">

>   Agent, embeddings and autocomplete models are not yet supported through graphical interface configuration. To enable them, you need to edit the configuration manually, see [Configure Continue through configuration file](#configure-continue-through-configuration-file).

Reviewer: ```suggestion
  Agents, embeddings and autocomplete models are not yet supported through graphical interface configuration. To enable them, you need to edit the configuration manually, see [Configure Continue through configuration file](#configure-continue-through-configuration-file).
```

</Message>

### Configure Continue through configuration file

To link Continue with Scaleway’s Generative APIs, you need to configure the settings file:

- Open your `config.yaml` settings file:

>   - If your already configured a **Local Assistant**, click on **Local Assistant** then on settings wheel icon to open your existing `config.yaml`

Reviewer: ```suggestion
  - If you have already configured a **Local Assistant**, click **Local Assistant** then click the **wheel icon** to open your existing `config.yaml`
```

  - Otherwise, create a `config.yaml` file inside your `.continue` directory.

> - Add the following configuration to enable Scaleway's Generative API. This configuration uses three different models for each tasks: 

Reviewer 2: ```suggestion
- Add the following configuration to enable Scaleway's Generative API. This configuration uses three different models for each task: 
```

>   - `devstral-small-2505` for agentic workflows through chat interface

Reviewer 2: ```suggestion
  - `devstral-small-2505` for agentic workflows through a chat interface
```

        provider: openai
        model: qwen2.5-coder-32b
        apiBase: https://api.scaleway.ai/v1/
        apiKey: ###SCW_SECRET_KEY###
        defaultCompletionOptions:
          maxTokens: 8000
          contextLength: 50000
        roles:
          - autocomplete
      - name: Embeddings Model - Scaleway
        provider: openai
        model: bge-multilingual-gemma2
        apiBase: https://api.scaleway.ai/v1/
        apiKey: ###SCW_SECRET_KEY###
        roles:
          - embed
        embedOptions:
          maxChunkSize: 256
          maxBatchSize: 32
    context:
      - provider: problems
      - provider: tree
      - provider: url
      - provider: search
      - provider: folder
      - provider: codebase
      - provider: web
        params:
          n: 3
      - provider: open
        params:
          onlyPinned: true
      - provider: docs
      - provider: terminal
      - provider: code
      - provider: diff
      - provider: currentFile
    ```
- Save the file at the correct location:
  - Linux/macOS: `~/.continue/config.yaml`
  - Windows: `%USERPROFILE%\.continue\config.yaml`
- In **Local Assistant**, click on **Reload config** or restart IntelliJ IDEA.

> Alternatively, a `config.json` file can be used with the following format. This format is however deprecated, and we recommend using `config.yaml` instead.

Reviewer 2: ```suggestion
Alternatively, a `config.json` file can be used with the following format. Note that this format is deprecated, and we recommend using `config.yaml` instead.
```

## Going further

> You can add additional parameters to configure your model behaviour by editing `config.yaml`.

Reviewer: ```suggestion
You can add more parameters to configure your model's behavior by editing `config.yaml`.
```
